### ISSUE TITLE
Cleanup old smartos machines after Data center migration

### ISSUE THREAD

**Issue by mhdawson** [nodejs/build#2035]
@cjihrig has confirmed that test release builds I sent him which used the new smartos machines look ok. @sam-github has confirmed that jobs looks ok. Time to remove the old smart os machines

- [x] remove from test ci firewall (@mhdawson)
- [x] remove from release ci firewall (@mhdawson)
- [x] remove from release secrets file (@mhdawson)
- [X] remove from test secrets file (@sam-github)
- [x] remove machines from test ci (@sam-github) 
- [x] remove machines from release ci (@mhdawson)
- [x] delete from us-east joyent datacenter (@mhdawson)
- [x] remove test machines from ansible inventory (@mhdawson) - https://github.com/nodejs/build/pull/2030
- [x] remove release machines from ansible inventory (@sam-github ) - - https://github.com/nodejs/build/pull/2036

**Comment by mhdawson** [comment]
@rvagg, @joaocgreis FYI, I've deleted the test/release smartos machines.  Believe you two are covering the rest of the remaining machines in the us-east datacenter.

**Comment by sam-github** [comment]
does freebsd not need migration?
```
      - joyent:                                                                    
          freebsd10-x64-1: {ip: 72.2.115.15}                                       
          freebsd10-x64-2: {ip: 72.2.114.23}  
```
they have the same subnet addrs as the smartos ones being removed

**Comment by richardlau** [comment]
> 
> 
> does freebsd not need migration?
> 
> ```
>       - joyent:                                                                    
>           freebsd10-x64-1: {ip: 72.2.115.15}                                       
>           freebsd10-x64-2: {ip: 72.2.114.23}  
> ```
> 
> they have the same subnet addrs as the smartos ones being removed

Weren't they removed in https://github.com/nodejs/build/pull/2012?

**Comment by sam-github** [comment]
No: https://github.com/nodejs/build/blob/09a0a64283c93c6b2ce4352f4605714a24e8b013/ansible/inventory.yml#L101-L102

**Comment by rvagg** [comment]
Still waiting on word from Joyent about their migration of the two infra server images. I haven't got around to testing the new ARM cross compile binaries for the already-moved servers that Joao did, but I'm not imagining there's a problem there so I haven't been treating that as urgent.

**Comment by joaocgreis** [comment]
The cross compile servers are done: https://github.com/nodejs/build/pull/2015. I left the 2 old machines shut down but not deleted. I can't get into the Joyent login portal now, so they might already be gone. I'll try again tomorrow and delete them if they're still there. Firewall and secrets are already done.

**Comment by mhdawson** [comment]
Closing as this issue was specific to the test/release smartos machines.
